CHANGELOG_UNRELEASED.md

```diff
@@ -1,3 +1,7 @@
 
 ### Changed:
-* Navigation UI has been redesigned with aesthetic improvements and better and more clear information about the route
\ No newline at end of file
+* Navigation UI has been redesigned with aesthetic improvements and better and more clear information about the route
+
+### Fixed:
+* Update user location when is in navigation mode and outside a route. Previously, if the user goes outside the route
+their location was not updated
```

Comment:
Fixed a bug detected in naviagation mode when the user goes outside the route. In previous versions while the user was outside the route the user location was not updated.

Eu darialle a volta, simplemente describiría o bug que había antes e así penso que se ve mellor o que se solucionou. Como o ves?